README.md

```diff
@@ -151,3 +153,7 @@ For example, for DBSCAN you get one of these print statements depending on which
 - `SKLEARNEX INFO: sklearn.cluster.DBSCAN.fit: fallback to original Scikit-learn`
 
 [Read more in the documentation](https://intel.github.io/scikit-learn-intelex/).
+
+## 🥇 Intel(R) oneAPI Base Toolkit
```

Comment:
I'm not sure that we need to write about base toolkit rather than AI toolkit. Also, I don't think that we need to extract this information in separate paragraph - better to put this in installation part.

Reply:
Thanks! Now I will deleted this and add this information later